README.md

```diff
@@ -156,7 +156,8 @@ export GOOGLE_CLOUD_PROJECT=dsp-artifact-registry
 export GAR_LOCATION=us-central1
 export GAR_REPOSITORY_ID=libs-release-standard
 gcloud auth login <you>@broadinstitute.org
-core/src/bin/publishRelease.sh```
+core/src/bin/publishRelease.sh
+```
```

Comment:
Fixed this as it was causing remaining below lines to appear in code block which wasn't necessary.